### PR TITLE
Fix fast jit issues

### PR DESCRIPTION
**Pull request by wenyongh**
And implement several opcodes

### CODE REVIEW

**Comment by lum1n0us** [comment]
LOL, we both made same modifications today.

**Comment by wenyongh** [comment]
> LOL, we both made same modifications today.

Yes, I tested several spec cases and made the changes :)

File: core/iwasm/fast-jit/fe/jit_emit_memory.c [hunk 1]
```diff
@@ -389,6 +389,10 @@ jit_compile_op_i64_store(JitCompContext *cc, uint32 align, uint32 offset,
         goto fail;
     }
 
+    if (jit_reg_is_const(value) && bytes < 8) {
```
Comment (lum1n0us) [inline core/iwasm/fast-jit/fe/jit_emit_memory.c:392]: what does this for?
if prefer to let the compiler do the job about const values, should skip the following `switch`, right?

File: core/iwasm/fast-jit/fe/jit_emit_numberic.c [hunk 2]
```diff
@@ -91,40 +91,143 @@
         PUSH_FLOAT(res);                     \
     } while (0)
 
+static inline uint32
```
[the remark this answers is not in the document]
Comment (lum1n0us) [reply core/iwasm/fast-jit/fe/jit_emit_numberic.c:94]: Plus, `popcnt` can return the count of number of bits set to 1

**Comment by lum1n0us** [comment]
LGTM

**Pull request merged**